code/game/objects/structures/divine.dm

```diff
@@ -6,19 +23,47 @@
 	anchored = TRUE
 	density = FALSE
 	can_buckle = 1
+	max_integrity = 30000
+	var/divine_chance = 5
 
-/obj/structure/sacrificealtar/on_attack_hand(mob/living/user, act_intent = user.a_intent, unarmed_attack_flags)
+/obj/structure/sacrificealtar/attackby(obj/item/I, mob/user)
 	. = ..()
+	var/divine = prob(divine_chance)
 	if(.)
 		return
+	if(istype(I, /obj/item/gun))
+		to_chat(user, "<span class='notice'>You attempt to sacrifice [I] by invoking the sacrificial ritual, it melts and breaks into pieces.</span>")
+		qdel(I)
+		message_admins("[ADMIN_LOOKUPFLW(user)] has sacrificed [key_name_admin(I)] on the sacrificial altar at [AREACOORD(src)].")
```

Comment:
I'd prefer this changed to a log, admemes get a lot of messages on average, and we're not really concerned about people deleting a dozen hipowers and knuckleguns, unless someone complains.

Reply:
Fair point. These messages are left over from the original version of the file. I think it's best to remove them altogether.